Firefox/index.js

```diff
@@ -1,115 +1,115 @@
-/* jshint moz: true */
+/* globals require */
 let self = require('sdk/self');
 let pageMod = require('sdk/page-mod');
 let prefs = require('sdk/preferences/service');
 let Request = require('sdk/request').Request;
 
 let scripts = [
-  'bridge.js',
-  'xkit/jquery.js',
-  'xkit/moment.js',
-  'xkit/nano.js',
-  'xkit/tiptip.js',
-  'xkit/xkit.js',
-  'xkit/editor.js',
-  'storage.js',
-  'bootstrap.js'
+	'bridge.js',
+	'xkit/jquery.js',
+	'xkit/moment.js',
+	'xkit/nano.js',
+	'xkit/tiptip.js',
+	'xkit/xkit.js',
+	'xkit/editor.js',
+	'storage.js',
+	'bootstrap.js'
 ].map(function(url) {
-  return self.data.url(url);
+	return self.data.url(url);
 });
 
 pageMod.PageMod({
-  include: '*.tumblr.com',
-  contentScriptFile: scripts,
-  contentStyleFile: self.data.url('xkit/xkit.css'),
-  contentScriptWhen: 'ready',
-  onAttach: onAttach
+	include: '*.tumblr.com',
+	contentScriptFile: scripts,
+	contentStyleFile: self.data.url('xkit/xkit.css'),
+	contentScriptWhen: 'ready',
+	onAttach: onAttach
 });
 
 let prefRoot = 'extensions.xkit7.';
 
 
 function onAttach(worker) {
-  let port = worker.port;
-  port.on('set', function(data) {
-    console.log('set(' + data.name + ', ' + data.value + ')');
-    prefs.set(prefRoot + data.name, data.value);
-  });
-  port.on('reset', function(data) {
-    console.log('reset(' + data.name + ')');
-    prefs.reset(prefRoot + data.name);
-  });
+	let port = worker.port;
+	port.on('set', function(data) {
+		console.log('set(' + data.name + ', ' + data.value + ')');
+		prefs.set(prefRoot + data.name, data.value);
+	});
+	port.on('reset', function(data) {
+		console.log('reset(' + data.name + ')');
+		prefs.reset(prefRoot + data.name);
+	});
 
-  port.on('reset_all', function(data) {
-    console.log('reset_all()');
-    let existingKeys = prefs.keys(prefRoot);
-    for (let name of existingKeys) {
-      prefs.reset(name);
-    }
-  });
+	port.on('reset_all', function(_data) {
```

Comment:
No, whoops, I meant to undo that change. I was tinkering with having unused function parameters be required to be named with a leading underscore, but I decided it wasn't that useful in the long run. Maybe in a followup PR.